client/src/component/common/MainButton/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './style.css';
+import { Button } from 'antd';
+import PropTypes from 'prop-types';
+
+const MainButton = ({
+  backgroundColor,
+  border,
+  color,
+  padding,
+  Buttontext,
+}) => (
+  <div>
```

Comment:
There's no need for this `div` to be here since `<Button>` already wrapping everything